Description: Set the default cgroup basedir
 When cgroup.conf is missing slurmd fails to start even if no cgroup plugin is
 specified. This patch make slurmd behave like cgroup.conf exists and is
 empty
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=14578
Last-Update: 2022-07-20

--- a/src/common/cgroup.c
+++ b/src/common/cgroup.c
@@ -297,6 +297,8 @@
 	if ((conf_path == NULL) || (stat(conf_path, &buf) == -1)) {
 		log_flag(CGROUP, "%s: No cgroup.conf file (%s)", __func__,
 			 conf_path);
+		slurm_cgroup_conf.cgroup_mountpoint =
+			xstrdup(DEFAULT_CGROUP_BASEDIR);
 		cg_conf_exist = false;
 	} else {
 		debug("Reading cgroup.conf file %s", conf_path);
